src/meta/meta_bulk_load_service.cpp

```diff
@@ -1120,6 +1160,7 @@ void bulk_load_service::on_partition_ingestion_reply(error_code err,
                  pid,
                  resp.err,
                  resp.rocksdb_error);
+        decrease_app_ingestion_count(pid);
```

Comment:
I understand it must `decrease_app_ingestion_count` if `err_code != ok`?  so you just:
```
if != ok {
 decrease_app_ingestion_count
}
```
and put it start line of this method